File: Usuario/backend/src/main/java/com/signspeak/backend/dto/DiccionarioSenaDTO.java
package com.signspeak.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DiccionarioSenaDTO {
    private Long idSena;
    private String nombre;  // ✅ Cambiado de "palabra" a "nombre"
    private String descripcion;
    private CategoriaSenaDTO categoria;
    private String urlVideo;
    private String urlImagen;
    private String urlAnimacion;
    private Integer duracionVideoSegundos;
    private String dificultad;  // Será "FACIL", "MEDIO", "DIFICIL"
    private Integer popularidad;
    private Integer vecesPracticada;
    private Boolean activo;
}

File: Usuario/mobile/app/src/main/java/com/signspeak/data/remote/dto/diccionario/DiccionarioSenaDTO.kt
package com.signspeak.data.remote.dto.diccionario

data class DiccionarioSenaDTO(
    val idSena: Long,
    val nombre: String,
    val descripcion: String?,
    val categoria: CategoriaSenaDTO?,
    val urlVideo: String?,
    val urlImagen: String?,
    val urlAnimacion: String?,
    val duracionVideoSegundos: Int?,
    val dificultad: String?, // "FACIL", "MEDIO", "DIFICIL"
    val popularidad: Int?,
    val vecesPracticada: Int?,
    val activo: Boolean
)
